po/fr.po

```diff
@@ -0,0 +1,15296 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: Comprehensive Rust 🦀\n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: 2023-04-11 19:31+0200\n"
+"Last-Translator: Olivier Charrez <olivier.charrez@hotmail.com>\n"
+"Language-Team: French <traduc@traduc.org>\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Generator: Poedit 3.2.2\n"
+
+#: src/SUMMARY.md:3
+msgid "Welcome to Comprehensive Rust 🦀"
+msgstr "Bienvenue à Rust Compréhensible 🦀"
+
+#: src/SUMMARY.md:4
+msgid "Running the Course"
+msgstr "Suivre le cours"
+
+#: src/SUMMARY.md:5
+msgid "Course Structure"
+msgstr "Structure du cours"
+
+#: src/SUMMARY.md:6
+msgid "Day 4"
+msgstr "Jour 4"
+
+#: src/SUMMARY.md:7
+msgid "Keyboard Shortcuts"
+msgstr "Raccourcis de clavier"
+
+#: src/SUMMARY.md:8
+msgid "Translations"
+msgstr "Traductions"
+
+#: src/SUMMARY.md:9
+msgid "Using Cargo"
+msgstr "Utiliser cargo"
+
+#: src/SUMMARY.md:10
+msgid "Rust Ecosystem"
+msgstr "Écosystème de Rust"
+
+#: src/SUMMARY.md:11
+msgid "Code Samples"
+msgstr "Exemples de codes"
+
+#: src/SUMMARY.md:12
+msgid "Running Cargo Locally"
+msgstr "Exécution locale"
+
+#: src/SUMMARY.md:15
+msgid "Day 1: Morning"
+msgstr "Jour 1 : Matin"
+
+#: src/SUMMARY.md:19 src/SUMMARY.md:76 src/SUMMARY.md:129 src/SUMMARY.md:185 src/SUMMARY.md:239
+msgid "Welcome"
+msgstr "Bienvenue"
+
+#: src/SUMMARY.md:20
+msgid "What is Rust?"
+msgstr "Qu'est-ce que Rust ?"
+
+#: src/SUMMARY.md:21
+msgid "Hello World!"
+msgstr "Bonjour le monde!"
+
+#: src/SUMMARY.md:22
+msgid "Small Example"
+msgstr "Petit exemple"
+
+#: src/SUMMARY.md:23
+msgid "Why Rust?"
+msgstr "Pourquoi Rust ?"
+
+#: src/SUMMARY.md:24
+msgid "Compile Time Guarantees"
+msgstr "Garanties de temps de compilation"
```

Comment:
This is not about guarantees on build times, it’s about guarantees that are provided already when compiling. How about « Garanties à la compilation »?